packages/xod-cli/src/messages.js

```diff
@@ -6,6 +6,13 @@ export default {
       'The error has no formatter, which is a bug. Report the issue to XOD developers.',
   }),
 
+  ARDUINO_CLI_ERROR: ({ code, message }) => ({
+    title: `'arduino-cli' error, code ${code}`,
+    note: message,
+    solution:
+      '`arduino-cli` error. The original tool output message is above. If you believe it is a bug, report the problem to XOD developers.',
```

Comment:
The `title` must not contain variables and should try to avoid punctuation. Solution should be a solution: not a doubt. So:

```js
title: 'Launch arduino-cli failed',
note: message,
solution: 'The error was not properly handled or formatted. This is a bug. Report it to XOD developers.'
```

Particular expected `arduino-cli` errors should be formatted accordingly: 'Compilation failed', 'Upload failed', etc (see IDE implementation)